fastlane/Fastfile

```diff
@@ -811,7 +815,8 @@ platform :ios do
            hosting_base_path,
            "--transform-for-static-hosting",
            "--enable-inherited-docs",
-           "--additional-symbol-graph-dir", ".build")
+           "--additional-symbol-graph-dir", ".build",
```

Comment:
⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ ⚠️ 

❗ This is broken right now. Its errors with error: Symbol with identifier 's:8Dispatch0A12TimeIntervalO' couldn't be found` and not really sure how to track it down

Temporary fix would be comment this ou but I don't know the side effect 🤷‍♂️